CRM/Financial/DAO/FinancialType.php

```diff
@@ -192,6 +193,7 @@ public static function &fields() {
           'title' => ts('Financial Type Is Active?'),
           'description' => ts('Is this property active?'),
           'where' => 'civicrm_financial_type.is_active',
+          'default' => '1',
```

Comment:
The BAO is actually defaulting to 0 at the moment - but that seems inconsistent with every other place we set a default for is_active and I propose switching to TRUE (removing the line in the BAO once this is merged)